_includes/components/gc-minister/gc-minister.html

```diff
@@ -0,0 +1,22 @@
+<section class="gc-minister">
+	<h2>{% if page.language == "en" %}Minister(s){% else %}Ministre(s){% endif %}</h2>
+	<div class="row wb-eqht-grd">
+		<div class="col-md-6">
+			<div class="row">
+				<div class="pull-right col-md-7 col-xs-12">
+					<h3><a href="#">{% if page.language == "en" %}The Honourable [Minister name]{% else %}L'honorable [prénom et nom de famille]{% endif %}</a></h3>
```

Comment:
Do they want a different column configuration for this one? May be it can take the full with but we do apply the content width limited rule?

Currently this item is wrapping on 2 lines and seems to be in a grid of 6 col.

Note that custom sizing will be only for when there is one minister, not when there is 2 or more.